src/autoComplete/index.tsx

```diff
@@ -206,16 +222,19 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
   }
 
   render() {
-    const shouldDisplayEmptyState = !this.hasMinCharsForSearch() && this.props.showList
-      && !isEmpty(this.props.renderEmptySearch)
+    const shouldDisplayEmptyState =
+      !this.hasMinCharsForSearch() && this.props.showList && !isEmpty(this.props.renderEmptySearch)
     const shouldDisplayBusyState = this.state.busy && this.props.showList
-    const shouldDisplayNoResults = this.hasMinCharsForSearch()
-      && !this.state.busy && this.state.noResults && this.props.showList
-    const shouldDisplayAutoCompleteList = this.hasMinCharsForSearch()
-      && !isEmpty(this.state.items) && !this.state.busy && this.props.showList
-    const listItems = shouldDisplayAutoCompleteList ? (
-      this.state.items
-     ) : this.props.renderEmptySearch
+    const shouldDisplayNoResults =
+      this.hasMinCharsForSearch() && !this.state.busy && this.state.noResults && this.props.showList
+    const shouldDisplayAutoCompleteList =
+      this.hasMinCharsForSearch() &&
+      !isEmpty(this.state.items) &&
+      !this.state.busy &&
+      this.props.showList
+    const listItems = shouldDisplayAutoCompleteList
```

Comment:
Maybe we should create a get method for this instead of having it in the render

Reply:
I created a refactoring ticket for that: BBCSPA-2041

Reply:
Deal